front/app/containers/InitiativesShow/LargerThanPhone.tsx

```diff
@@ -57,9 +57,6 @@ import InitiativeBanner from './InitiativeBanner';
 const Container = styled.main`
   display: flex;
   flex-direction: column;
-  min-height: calc(
-    100vh - ${(props) => props.theme.menuHeight + props.theme.footerHeight}px
-  );
```

Comment:
This seems not needed, comments were appearing way down on proposal pages with little content.